terraform/.env.template

```
@@ -0,0 +1 @@
+CLOUDFLARE_API_TOKEN=op://Private/Cloudflare API Token mf-dashboard/credential
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)